src/commands/link.ts

```diff
@@ -61,7 +49,7 @@ export const data = new SlashCommandBuilder()
       .setName("topic")
       .setDescription("The topic to link to")
       .setRequired(true)
-      .addChoices(Object.keys(LINKS).map((k) => [k, k]))
+      .addChoices(Object.keys(LINKS).map((k) => [LINKS[k].description, k]))
```

Comment:
Looks like the key is used to select.
So `/link topic:kata` shows two entries for example, and it's a bit annoying.

Maybe use a unique prefix + description as key?

```typescript
.addChoices(Object.keys(LINKS).map((k) => [`${k}: ${LINKS[k].description}`, k]))
```

Reply:
Wait, that might not help.